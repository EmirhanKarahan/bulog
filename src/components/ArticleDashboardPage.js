import React from "react";
import ArticleList from "./ArticleList";

export default function ArticleDashboardPage() {
  return <ArticleList></ArticleList>;
}
