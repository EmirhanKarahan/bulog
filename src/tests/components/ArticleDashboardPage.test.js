import React from "react";
import { shallow } from "enzyme";
import ArticleDashboardPage from "../../components/ArticleDashboardPage";

test("should render ArticleDashboardPage correctly", ()=>{
    const wrapper = shallow(<ArticleDashboardPage />)
    expect(wrapper).toMatchSnapshot();
})