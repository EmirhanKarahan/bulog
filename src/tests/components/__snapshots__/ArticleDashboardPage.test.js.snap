// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ArticleDashboardPage correctly 1`] = `<Connect(ArticleList) />`;
